### ISSUE TITLE
Cypress does not show or strips the "cause" property in the caught exception

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#25449]
### What would you like?

The new JS standard allows to set the `cause` when throwing an error

```js
throw new Error('Top level error', { cause: new Error('inner problem') })
```

Cypress catches the error (for example in `uncaught:exception`), but it strips the `cause` property I think so I cannot see it. It would be nice to show the `cause` automatically.

### Why is this needed?

Better understanding the application errors when they are thrown

### Other

My repo showing it in https://github.com/bahmutov/cypress-exception-causeshows

**Comment by emilyrohrbough** [comment]
@bahmutov this request seems reasonable for surfacing the root cause of an error. Is this a contribution you would be interested in contributing?

**Comment by bahmutov** [comment]
No. Sent from my iPhoneOn Mar 28, 2023, at 16:03, Emily Rohrbough ***@***.***> wrote:﻿
@bahmutov this request seems reasonable for surfacing the root cause of an error. Is this a contribution you would be interested in contributing?
